entrypoint.sh

```diff
@@ -36,9 +39,30 @@ apt -y install $APT_DEPENDENCIES
 
 mkdir build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=coverage
+
+echo "SCRIPT_BEFORE_CMAKE"
+if [ ! -z "$SCRIPT_BEFORE_CMAKE" ] ; then
+  bash $SCRIPT_BEFORE_CMAKE
+fi
+
+if [ ! -z "$CODECOV_TOKEN" ] ; then
+  cmake .. -DCMAKE_BUILD_TYPE=coverage
+else
+  cmake ..
+fi
+
+echo "SCRIPT_BETWEEN_CMAKE_MAKE"
+if [ ! -z "$SCRIPT_BETWEEN_CMAKE_MAKE" ] ; then
+  bash $SCRIPT_BETWEEN_CMAKE_MAKE
+fi
+
 make
 
+echo "SCRIPT_AFTER_MAKE"
+if [ ! -z "$SCRIPT_AFTER_MAKE" ] ; then
+  bash $SCRIPT_AFTER_MAKE
+fi
+
 export CTEST_OUTPUT_ON_FAILURE=1
 make test
```

Comment:
I want to run `make test` before `make install`, so I'd want an extra step after `make test` for `make install` and examples compilation

Reply:
27dc5baefafb196fc2fd48353f4e6c03e22e3cfb